fetch_depth_layer/include/fetch_depth_layer/depth_layer.h

```diff
@@ -100,7 +113,8 @@ class FetchDepthLayer : public VoxelLayer
 
   // retrieves depth image from head_camera
   // used to fit ground plane to
-  ros::Subscriber depth_image_sub_;
+  message_filters::Subscriber<sensor_msgs::Image> depth_image_sub_;
+  boost::shared_ptr< tf::MessageFilter<sensor_msgs::Image> > depth_image_filter_;
```

Comment:
Ah, never mind, it looks like this was for #37 and then got included in this one also.